src/routes/anime/explore/+page.svelte

```diff
@@ -242,7 +241,7 @@
 							role="button"
 							class="peer placeholder w-full rounded-lg border-none bg-neutral py-3 text-base font-semibold leading-none text-neutral-content placeholder:font-medium focus:ring-0 md:w-[11vw] md:rounded-[0.5vw] md:bg-neutral md:py-[0.8vw] md:pl-[1vw] md:text-[1vw]"
 						/>
-						{#if !is_empty(selected_items ?? [])}
+						{#if (selected_items ?? []).length}
```

Comment:
```
selected_items =  []
```

Adding ` ?? [] ` is bad design

Reply:
already fixed in https://github.com/baseplate-admin/coreproject-v4-ui/pull/477/commits/5051c7993828a8b152a3e2d2badb6902817f99f5